### ISSUE TITLE
Side nav list STEP 1-10: steps missing alt text for (!) and checkmark icons and missing state for selected step

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#826]
The side nav list steps links have error status icons and checkmark status icons that have no text alternative for screen reader users. 
There is also no indication to screen readers which is the current step (marked visually with the thick green left border). 
 
Use image alt text techniques like role="img" and aria-label.
http://w3c.github.io/aria/aria/aria.html#img
http://pauljadam.com/demos/img.html
 Use aria-current for the current step. http://w3c.github.io/aria/aria/aria.html#aria-current
https://tink.uk/using-the-aria-current-attribute/ 

![image](https://user-images.githubusercontent.com/28604890/30260828-9863ca04-9697-11e7-8788-21f5fc5c6077.png)

WCAG: 1.1 Text Alternatives - 1.1.1 Non-text Content 
EPERM-129

**Comment by npage123** [comment]
- [x] Check with Mac Voice Over
- [x] Check with JAWS
